collector/src/main/java/org/apache/hertzbeat/collector/collect/http/HttpCollectImpl.java

```diff
@@ -133,7 +133,7 @@ public void collect(CollectRep.MetricsData.Builder builder,
             // Option 1: Parse using InputStream, but this requires significant code changes;
             // Option 2: Manually trigger garbage collection, similar to how it's done in Dubbo for large inputs.
             String resp = EntityUtils.toString(response.getEntity(), StandardCharsets.UTF_8);
-            if (StringUtils.hasText(resp)) {
+            if (!StringUtils.hasText(resp)) {
                 log.info("http response entity is empty, status: {}.", statusCode);
```

Comment:
When `resp` is empty, should we return directly without executing the subsequent logic?

Reply:
some http request may not has response data themselves, but still need to be processed.